frontend/ui/elements/screensaver_menu.lua

```diff
@@ -98,29 +99,23 @@ return {
         end
     },
     {
-        text = _("Use message as screensaver"),
+        text = _("Leave screen as it is"),
         checked_func = function()
-            if screensaverType() == "message" or screensaverType() == nil then
+            if screensaverType() == "disable" then
                 return true
             else
                 return false
             end
         end,
         callback = function()
-            G_reader_settings:saveSetting("screensaver_type", "message")
+            G_reader_settings:saveSetting("screensaver_type", "disable")
         end
     },
     {
-        text = _("Leave screen as it is"),
-        checked_func = function()
-            if screensaverType() == "disable" then
-                return true
-            else
-                return false
-            end
-        end,
+        text = _("Add message to screensaver"),
```

Comment:
I guess you can add a `separator = true,` to the previous menu item - and may be also to this one (if it needs to be separated from the following "Settings" - or not if we consider it a bit like a setting).

Reply:
Just tried it and it looks better with separators above and below. Will commit that.